src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -65,6 +65,35 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(SUCCESS_MESSAGE);
     }
 
+    /**
+     * Parses a single tag from its JSON representation to its CSV representation.
+     * e.g. {\n \"friends\" : null\n} -> friends
+     * @param tagString
+     * @return a single parsed tag
+     */
+    static String parseTags(String tagString) {
+        // Remove leading and trailing whitespace
+        tagString = tagString.trim();
+
+        // Check if the string starts with { and ends with }
+        if (tagString.startsWith("\"{") && tagString.endsWith("}\"")) {
+            // Remove the outer braces
+            tagString = tagString.substring(DISTANCE_TO_TAG, tagString.length() - 1);
+
+            // Split by : and take the first part
+            String[] parts = tagString.split(":");
+            if (parts.length > 0) {
+                // Trim and remove quotes from the tag name
+                String trimmed = parts[0].trim()
+                        .replaceAll("\"", "")
+                        .replaceAll("\\\\", "");
+                return trimmed;
+            }
+        }
+        // If the format doesn't match, return the original string
+        return tagString;
+    }
+
```

Comment:
Looks good and clean. I liked how you handled this removal unnecessary characters when processing each tag.